iml-manager-cli/src/stratagem.rs

```diff
@@ -86,6 +92,43 @@ pub struct StratagemScanData {
     purge_duration: Option<u64>,
 }
 
+#[derive(serde::Serialize, StructOpt, Debug)]
+pub struct StratagemFilesyncData {
+    /// action, either push or pull
+    #[structopt()]
+    action: String,
+    /// The name of the filesystem to scan
+    #[structopt(short = "f", long = "filesystem")]
+    filesystem: String,
+    /// The remote filesystem
+    #[structopt(short = "r", long = "remote")]
+    remote: String,
+    /// Match expression
+    #[structopt(short = "e", long = "expression")]
+    expression: String,
+    #[structopt(skip = true)]
+    filesync: bool,
+}
+
+#[derive(serde::Serialize, StructOpt, Debug)]
+pub struct StratagemCloudsyncData {
+    /// action, either push or pull
+    #[structopt()]
```

Comment:
Can we put params in here?

Reply:
Seems like this should be an enum.

Reply:
Not sure what you mean, could you please clarify?